docs/reference.md

```diff
@@ -1971,6 +1971,165 @@ str_field:
   ]
 ```
 
+## Apache ORC files
```

Comment:
Could you move this ORC section to the bottom of the reader/writers section?
i.e.

- Parquet files
- IPC files
- IPC streams
- ORC files

ORC should be lower priority than the others
Also, you need to add the ORC functions to the table of contents at the top of `reference.md`

Reply:
moving